site/src/lib/api/handler/apiKeyRequiredHandler.ts

```diff
@@ -10,4 +10,4 @@ export const createApiKeyRequiredHandler = <
   RequestBody = unknown,
   Response = unknown,
 >(): NextConnect<BaseApiRequest<RequestBody>, BaseApiResponse<Response>> =>
-  createBaseHandler().use(hasValidApiKeyMiddleware);
+  createBaseHandler({ isPublicApi: true }).use(hasValidApiKeyMiddleware);
```

Comment:
The assumption I'm making here is that all APIs which need an API key to function should be accessible by external clients.